playbooks/vagrant-devstack.yml

```diff
@@ -17,6 +17,7 @@
     edxapp_use_custom_theme: true
     edxapp_theme_name: 'lagunita'
     edxapp_theme_source_repo: 'https://github.com/Stanford-Online/lagunita-theme.git'
+    ENABLE_ELK: true
```

Comment:
@caesar2164 It's literally a variable to configure it ;)

We have to default it to something;
you disable it by setting `ENABLE_ELK=false`.